lib/traject/readers/folio_postgres_reader.rb

```diff
@@ -254,6 +276,17 @@ def sql_query(conditions, addl_from: nil)
                     'instructorObjects', cl.jsonb #> '{instructorObjects}'
                   )
                 ) FILTER (WHERE cc.id IS NOT NULL),
+              '[]'::jsonb),
+            'requests',
```

Comment:
🤔 If we always have an item, could we nest this under the item (like we're doing with bound-withs and the holdings record)?

Reply:
I think we could. I'm not sure how to write that query though.